nesta/packages/crunchbase/crunchbase_collect.py

```diff
@@ -69,8 +69,11 @@ def get_files_from_tar(files, nrows=None):
     dfs = []
     with crunchbase_tar() as tar:
         for filename in files:
```

Comment:
previously all dealings with `null` values from the CSV were dealt with in individual functions. I've refactored so that a consistent procedure is performed for all crunchbase CSVs